content/PreparedStatement/PreparedStatement.q.md

```diff
@@ -0,0 +1,1677 @@
+    }
+?>
+```
+
+```php
+// student.php
+<?php
+    require_once("connection.php");
+
+    try {
+        $sql = "UPDATE students SET gender= ? WHERE gender= ?";
+        $pstmt = $conn->prepare($sql);
+        if (!$pstmt->execute(array('M', 'Male'))) {
+            throw new Exception("Unable to update values in the table.");
+        }
+        echo "Successfully updated values in the table.";
+    } catch (Exception $e) {
+        echo $e->getMessage();
+    }
+?>
+```
+/// type=SS, answer=[5]
+
+Execute the program. What is its output?
+
+- It produces an error.
+
+- No output is displayed.
+
+- It prints `Unable to establish a connection.`.
+
+- It prints `Successfully connected to the database.` and `Unable to update values in the table.`.
+
+- It prints `Successfully connected to the database.` and `Successfully updated values in the table.`.
+
+
+/// type=MS, answer=[3,5]
+
+Which statements correctly describe `UPDATE students SET gender= ? WHERE gender= ?` on line 5 of `student.php`?
+
+- It modifies all rows in the `gender` column.
+
+- It selects `students` as the table to be deleted.
+
+- It selects `students` as the table to be modified.
+
+- It updates the rows in the `gender` column with the value substituted by the named placeholder `?`.
+
+- It updates the rows in the `gender` column with the value substituted by the positional placeholder `?`.
+
+
+/// type=MS, answer=[1,3,5]
+
+Which statements correctly describe `$pstmt->execute(array('M', 'Male'))` on line 7 of `student.php`?
+
+- It returns the boolean value `true`.
+
+- It holds the values `M` and `Male` in the SQL statement.
+
+- It calls the `execute()` method of the `$pstmt` object.
+
+- It assigns the specified values `M` and `Male` to the SQL statement.
+
+- It executes the argument `array('M', 'Male')` in the `execute()` method.
+
+
+:::
+
+
+:::
+
+/// type=REPL, filename=[connection.php,student.php]
+
+```php
+// connection.php
+<?php
+    $host = 'localhost';
+    $db = 'LibraryDB';
+    $port = '5432';
+    $username = 'postgres';
+    $password = 'Admin01';
+
+    try {
+        $conn = new PDO("pgsql:host=$host;port=$port;dbname=$db;user=$username;password=$password");
+        if ($conn) {
+            echo "Successfully connected to the database. <br />";
+        }
+    } catch (Exception $e) {
+        echo "Unable to establish a connection."; 
+    }
+?>
+```
+
+```php
+// student.php
+<?php
+    require_once("connection.php");
+
+    $pstmt = $conn->prepare("SELECT * FROM students WHERE gender = ?");
+    $pstmt->execute(array('Male'));
+    $data = $pstmt->fetchAll();
+    
+    foreach ($data as $row) {
+        echo $row['student_id'].$row['first_name'].$row['last_name'].$row['birth_date'].$row['gender']. ' ';
+    }
+?>
+```
+/// type=SS, answer=[4]
+
+The statement `$pstmt->execute(array('Male'));` on line 5 of `student.php`, replace the value `Male` with `M`. Execute the program. What is its output?
```

Comment:
Basic grammar error here...

Missing `In` at the beginning of the statement. Please check. Thanks.

> __In__ the statement...